app/models/car.rb

```diff
@@ -0,0 +1,12 @@
+class Car < ApplicationRecord
+  belongs_to :user
```

Comment:
Kindly add these associations to allow for conformity of our app;

class Car < ApplicationRecord
  has_many :reservations, dependent: :destroy
  has_many :users, through: :reservations
  validates :name, presence: true, length: { in: 3..25 }
  validates :brand, presence: true, length: { in: 3..25 }
  validates :image, presence: true
  validates :lending_fee, presence: true, length: { in: 1..1000 }, numericality: { only_integer: true }
  validates :description, presence: true, length: { in: 5..1000 }
end

## You may remove the categories resources